pkg/cmd/experimental/syncgroups/ad/augmented_ldapinterface.go

```diff
@@ -60,6 +62,7 @@ func (e *AugmentedADLDAPInterface) GroupEntryFor(ldapGroupUID string) (*ldap.Ent
 		return nil, err
 	}
 
+	fmt.Printf("Augmented AD: query for group entry %q\n", ldapGroupUID)
```

Comment:
You can't output this here.  Anything going to system out will mess up piping ability.